chia/server/server.py

```diff
@@ -42,6 +42,13 @@
 max_message_size = 50 * 1024 * 1024  # 50MB
 
 
+class SamePeerIdError(Exception):
```

Comment:
Maybe `class SamePeerIdError(ServerException)` in `chia.utils.errors`?

Reply:
Something in that direction sounds reasonable.  I don't see a `ServerException` there though.  Maybe still in an unmerged PR?

Reply:
>I don't see a `ServerException` there though. Maybe still in an unmerged PR?

I thought i added it at some point. But yes, then maybe just add it and if i have a PR where i add it i can remove it when i revisit it :)